promenade/templates/common/etc/apt/apt.conf

```diff
@@ -0,0 +1,9 @@
+#Promenade Generated
+{% if config['Network']['http_proxy'] is defined and config['Network']['https_proxy'] is defined %}
+Acquire::http::Proxy "{{ config['Network']['http_proxy'] }}";
+Acquire::https::Proxy "{{ config['Network']['https_proxy'] }}";
+{% elif config['Network']['http_proxy'] is defined %}
```

Comment:
I think it might be simpler to do something like:

```
{% if config['Network']['http_proxy'] is defined %}
Acquire::...
{% endif %}
{% if config['Network']['https_proxy'] is defined %}
Acuquire...
{% endif %}
```